lib/matplotlib/widgets.py

```diff
@@ -1742,16 +1661,10 @@ def _clicked(self, event):
         coords = self._buttons.get_offset_transform().transform(
             self._buttons.get_offsets())
         distances = {}
-        if hasattr(self, "_circles"):  # Remove once circles is removed.
-            for i, (p, t) in enumerate(zip(self._circles, self.labels)):
-                if (t.get_window_extent().contains(event.x, event.y)
-                        or np.linalg.norm(pclicked - p.center) < p.radius):
-                    distances[i] = np.linalg.norm(pclicked - p.center)
-        else:
-            for i, t in enumerate(self.labels):
-                if (i in inds["ind"]
-                        or t.get_window_extent().contains(event.x, event.y)):
-                    distances[i] = np.linalg.norm(pclicked - coords[i])
+        for i, t in enumerate(self.labels):
+            if (i in inds["ind"]
+                    or t.get_window_extent().contains(event.x, event.y)):
```

Comment:
Fits in a line now:
```suggestion
            if i in inds["ind"] or t.get_window_extent().contains(event.x, event.y):
```

Reply:
I have a further PR that rewrites this loop, so let's just get this in for now.